test/util/benchmark_results.cpp

```diff
@@ -369,10 +369,10 @@ void BenchmarkResults::save_results()
     baseline_file += ".baseline";
     int r;
     if (!util::File::exists(baseline_file)) {
-        r = link(name.c_str(), baseline_file.c_str());
+        link(name.c_str(), baseline_file.c_str());
```

Comment:
CI now has a warning: `ignoring return value of 'int link(const char*, const char*)', declared with attribute warn_unused_result [-Wunused-result]` Maybe there's a better way to fix it?